node-js/node-js.md

```diff
@@ -20,6 +20,7 @@ This lab exercise will introduce Node.js, a powerful tool that lets you run Java
 
 ### Materials
 
+- [The beginner's guide: Understanding Node.js & Express.js fundamentals](https://medium.com/@LindaVivah/the-beginners-guide-understanding-node-js-express-js-fundamentals-e15493462be1)
```

Comment:
Hi Nazreen, how long does this take? Can you add the time next to it, and add that to the the Projected Time?